src/defines.cpp

```diff
@@ -1,117 +0,0 @@
-/**
- * @file defines.cpp
- * Manages the table of defines for some future time when these will be used to
- * help decide whether a block of #if'd code should be formatted.
- *
- * !! This isn't used right now. !!
- *
- * @author  Ben Gardner
- * @license GPL v2+
- */
-#include "defines.h"
-#include "uncrustify_types.h"
-#include "char_table.h"
-#include "args.h"
-#include <cstring>
-#include <cerrno>
-#include <cstdlib>
-#include <map>
-#include "unc_ctype.h"
-#include "chunk_list.h"
-#include "prototypes.h"
-
-using namespace std;
-
-typedef map<string, string> defmap;
-defmap defines;
```

Comment:
Hmm... I don't see any way this could even be accessed outside this file. That's pretty good evidence of non-usefulness.